"Do not know how to load path" error in Storybook 8

Is this addon compatible with Storybook 8? I upgraded and am getting:

```
Failed to scan for dependencies from entries:
  [path]/src/stories/Demo.stories.vue

  ✘ [ERROR] Do not know how to load path: html:[path]/src/stories/Demo.stories.vue?vue&type=stories
```

--- a/src/core/logger.ts
+++ b/src/core/logger.ts
@@ -1,3 +1,3 @@
 import { consola } from 'consola'
 export const logger = consola.withTag('[storybook:vue]')
-// logger.level = LogLevel.Debug
+// logger.level = 4 // Debug

--- a/src/index.ts
+++ b/src/index.ts
@@ -22,8 +22,8 @@ export const unpluginFactory: UnpluginFactory<Options | undefined> = (
         ...options,
       })) as { id: string; external: boolean }
 
-      // If it cannot be resolved or is external, just return it so that Rollup can display an error
-      if (!resolution || resolution.external) return resolution
+      // If it cannot be resolved or is external, return undefined so that the next plugin can handle it
+      if (!resolution || resolution.external) return undefined
 
       // We append a custom "type" so that the vue plugin is not handling the import
       resolution.id = resolution.id + STORIES_INTERNAL_SUFFIX

